datahub/main.py

```diff
@@ -123,3 +124,24 @@ def get_dsr_data(  # type: ignore[misc]
     filtered_data = dt.dsr_data[start : end + 1 if end else end]
 
     return {"data": filtered_data}
+
+
+@app.get("/wesim")
+def get_wesim_data() -> dict[Hashable, Any]:  # type: ignore[misc]
+    """GET method function for getting Wesim data as JSON.
+
+    Returns:
+        A Dict containing the Wesim Dataframes
+    """
+    dt.wesim_data = get_wesim()
+
+    data = {
+        "Capacity": dt.wesim_data["Capacity"].to_dict(orient="split"),
+        "Regions": dt.wesim_data["Regions"].to_dict(orient="split"),
+        "Interconnector Capacity": dt.wesim_data["Interconnector Capacity"].to_dict(
+            orient="split"
+        ),
+        "Interconnectors": dt.wesim_data["Interconnectors"].to_dict(orient="split"),
+    }
```

Comment:
A little frustrating we have to do this `to_dict` method on each of them, but I see why it's needed.

It might be better to just edit `get_wesim` to save `dt.wesim_data` with this format - there is no reason I see for why it needs to stay in memory as a pandas DataFrame. And this way we don't need to run the `to_dict` method every time the API is called.